spec/support/test_helpers/cycle_timetable_helper.rb

```diff
@@ -0,0 +1,39 @@
+module CycleTimetableHelper
```

Comment:
I think this is great. One question is whether it's safe to call any of these methods twice in the same test? I guess you'd get different (random) results? I wonder if it's useful to memoize the results per spec, i.e. memoize all these methods but reset the state `after` each.

Reply:
That's a really good point Steve. I'll sort that now!